atomic-structure/content.md

```diff
@@ -0,0 +1,25 @@
+# Radioactive Decay
+#### Learning objective
+Basic understanding of atomic structure
+
+How atomic structure relates to radioactivity
+___
+#### Overview
+The most straightforward conceptualization of the atom is the shell model, first proposed by Niels Bohr.
+
+The nucleus, containing positively charged protons and neutrons, which carry no charge, is surrounded by shells of negatively charged electrons. Each shell can only hold a fixed number of electrons, and each shell essentially represents a principal energy level. The electrons orbit around the nucleus. 
```

Comment:
For readability, I recommend "... containing positively-charged protons and uncharged neutrons, is surrounded by shells of negatively-charged electrons"